src/Tickets/RSVP/V2/Repositories/Attendee_Repository.php

```diff
@@ -0,0 +1,275 @@
+<?php
+/**
+ * V2 Attendee Repository for TC-RSVP attendees.
+ *
+ * @since TBD
+ *
+ * @package TEC\Tickets\RSVP\V2\Repositories
+ */
+
+namespace TEC\Tickets\RSVP\V2\Repositories;
+
+use TEC\Tickets\Commerce\Attendee;
+use TEC\Tickets\Event;
+use TEC\Tickets\Repositories\Traits\Get_Field;
+use TEC\Tickets\RSVP\Contracts\Attendee_Privacy_Handler;
+use Tribe__Repository;
+use Tribe__Repository__Interface;
+use WP_Post;
+
+/**
+ * Class Attendee_Repository
+ *
+ * Repository for querying TC-RSVP attendees.
+ * Extends the base repository and provides filters for RSVP-specific queries
+ * including going/not-going status.
+ *
+ * @since TBD
+ *
+ * @package TEC\Tickets\RSVP\V2\Repositories
+ */
+class Attendee_Repository extends Tribe__Repository implements Attendee_Privacy_Handler {
+	use Get_Field;
+
+	/**
+	 * RSVP status meta key.
+	 *
+	 * @since TBD
+	 *
+	 * @var string
+	 */
+	public const RSVP_STATUS_META_KEY = '_tec_tickets_commerce_rsvp_status';
+
+	/**
+	 * The unique fragment that will be used to identify this repository filters.
+	 *
+	 * @since TBD
+	 *
+	 * @var string
+	 */
+	protected $filter_name = 'tc_rsvp_attendees';
+
+	/**
+	 * Constructor.
+	 *
+	 * @since TBD
+	 */
+	public function __construct() {
+		parent::__construct();
+
+		// Set the post type to TC attendees.
+		$this->default_args['post_type']   = Attendee::POSTTYPE;
+		$this->default_args['post_status'] = 'publish';
+
+		// Set up schema for filtering.
+		$this->schema = array_merge(
+			$this->schema,
+			[
+				'event'     => [ $this, 'filter_by_event' ],
+				'ticket'    => [ $this, 'filter_by_ticket' ],
+				'going'     => [ $this, 'filter_by_going' ],
+				'not_going' => [ $this, 'filter_by_not_going' ],
+			]
+		);
+
+		// Add simple meta schema entries.
+		$this->add_simple_meta_schema_entry( 'event_id', Attendee::$event_relation_meta_key );
+		$this->add_simple_meta_schema_entry( 'ticket_id', Attendee::$ticket_relation_meta_key );
+		$this->add_simple_meta_schema_entry( 'user_id', Attendee::$user_relation_meta_key );
+		$this->add_simple_meta_schema_entry( 'rsvp_status', self::RSVP_STATUS_META_KEY );
+		$this->add_simple_meta_schema_entry( 'checked_in', Attendee::$checked_in_meta_key );
```

Comment:
Are we going to be able to specify `->set_args( [ 'checked_in' => true ] )` if the checked_in key is not passed to the update_field_aliases ?

Similar for other entries.